activitysim/core/configuration/top.py

```diff
@@ -585,6 +585,18 @@ class Settings(PydanticBase, extra="allow", validate_assignment=True):
         compatible with using :py:attr:`Settings.sharrow`.
     """
 
+    omx_ignore_patterns: list[str] = []
+    """
+    List of regex patterns to ignore when reading OMX files.
+
+    This is useful if you have tables in your OMX file that you don't want to
+    read in.  For example, if you have both time-of-day values and time-independent
+    values (e.g., "BIKE_TIME" and "BIKE_TIME__AM"), you can ignore the time-of-day
+    values by setting this to ["BIKE_TIME__.+"].
```

Comment:
Is this required for sharrow? If so, I think we need to put an assert statement in the skim read for sharrow somewhere that will prevent this from happening and provide some info to the user about the error.

Reply:
Yes, not for sharrow _per se_, but the `SkimDataset` converts all the by-time-of-day skim variables into three dimensional arrays (o,d,time).  So without this, in the example the OMX files essentially have two different sets of data for the same named variable, one two-dimension `BIKE_TIME`, and one three-dimension `BIKE_TIME`, which coincidentally happens to have no variance across the temporal dimension. We can't have two variables in the same namespace with the same name, so one is overwritten.

Reply:
Opened an issue #873 to address this.